### PR TITLE
metrics support changes to azure-support

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* Fixed website metrics which was not working.
* Made dynamic filters for metrics based on providers.

### CODE REVIEW

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts [hunk 1]
```diff
@@ -160,6 +164,13 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
     }
   }
 
+  setFilters(){
+    // we are targeting the second object in the FormFields where values for AGGREGATION is there
+    this.formFields[2].options = ['Average', 'Total', 'Maximum']
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: Is this accurate? We are seeing mixed case here. Do we need to set both value and values?

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts [hunk 2]
```diff
@@ -447,7 +459,9 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
         break;
       case 'website':
         let websiteAssets = _(this.queryDataRaw.assets).map('type').uniq().value();
-        this.filters.addField('Filter By:', 'ASSET', websiteAssets, 'select', null, 'cloudfront');
+        if (this.provider != 'azure'){
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: Will this work if we add another provider. Should we just say `this.provider == 'aws'` instead?

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts (hunk 1 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: value and values both are necessary as the filter modal component being used there needs both of these fields

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts (hunk 2 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: My Bad. Changed to provider == 'aws'

**Pull request merged**